Tasks/Common/webdeployment-common/jsonvariablesubstitutionutility.ts

```diff
@@ -64,6 +64,66 @@ export function substituteJsonVariable(jsonObject, envObject) {
     }
 }
 
+function stripJsonComments(content) {
```

Comment:
Add L0, with single line, multi-line and some huge comments.

Reply:
sure. planned to add them once I resolve the comments.

Reply:
Added L0 Tests